developer-docs-site/static/api/latest-api.html

```diff
@@ -11,11 +11,10 @@
   </head>
   <body>
     <elements-api id="spec_viewer"
-      apiDescriptionUrl="https://raw.githubusercontent.com/aptos-labs/aptos-core/main/api/doc/spec.yaml"
+      apiDescriptionUrl="https://raw.githubusercontent.com/aptos-labs/aptos-core/cbbca042910882ed335420ee44dca0e7588d0094/api/doc/spec.yaml"
```

Comment:
why peg this to a specific revision when it says "latest"?

Reply:
Yeah I would keep this as the latest